### ISSUE TITLE
Migration from an old project with amplify-cli to amplify-js 

### ISSUE THREAD

**Issue by michaelcuneo** [aws-amplify/amplify-js#2551]
I have a few projects using the old amplify-cli

I would like to migrate them all across to using the Amplify Console, and Amplify-JS... I am creating new projects with amplify-js, but how do I get my old ones to migrate across without any issues, would I need to recreate the S3 buckets for storage, databases, etc... or just initialise it with it's existing resources somehow?

**Comment by haverchuck** [comment]
@michaelcuneo -  If you are just trying to deploy your front-end with the Amplify Console, it sounds like you just need to disable the backend build as described [here](https://docs.aws.amazon.com/amplify/latest/userguide/getting-started.html) (Section 3).

**Comment by michaelcuneo** [comment]
So if I just keep a record of the amplify-cli generated config... reinit the project with amplify-js client, then switch the config over... it will work?

**Comment by haverchuck** [comment]
@michaelcuneo - Are you saying that you want to use the Amplify Console to manage your frontend and backend resources (the ones created via the CLI), or that you want to simply manage your frontend with the Amplify Console and have said frontend use the resources originally created via the CLI, and have those resources remain unchanged?  

Also, when you say the 'old amplify-cli', do you mean the awsmobile cli?

**Comment by michaelcuneo** [comment]
@haverchuck  Yeah, I've got an old project using awsmobile-cli and it's very slow... I forgot what it was called, and used the name wrong... was going to refactor the entire login authManagement system, and get it all set up under the new amplify-js client. There are only test users and test data in there right now, but I needed to migrate it over easily without breaking the production. I guess I can just duplicate the entire project to a new project. Then transfer dev to prod after fixing it all up, but I just thought maybe there's an easier way to delete the old awsmobile folders or quickly adjust the config to suit amplify-cli instead...
